charts/datadog/values.yaml

```diff
@@ -976,6 +992,12 @@ clusterChecksRunner:
   additionalLabels: {}
     # key: "value"
 
+  # clusterChecksRunner.securityContext -- Allows you to overwrite the default PodSecurityContext on the clusterchecks pods.
+  securityContext: {}
+
+  # clusterChecksRunner.containerSecurityContext -- Allows you to overwrite the default container SecurityContext on the clustercheck containers.
+  containerSecurityContext: {}
```

Comment:
I'm not sure the `containerSecurityContext` fields are useful as we only have 1 container / pod ATM, it may add confusion. We will add them if we need to when we put multiple containers in these pods.

Reply:
Though my need is strictly to set capabilities on the `system-probe` container, I've added these to resolve https://github.com/helm/charts/pull/17274#issuecomment-538536277 as discussed between @coreypobrien and @CharlyF when the conversation had originally spun from the `system-probe` container issue.

Reply:
Yeah I agree with this. So having the `containerSecurityContext` in all containers is fine, though currently in `clusterAgent` and `clusterChecksRunner`, as we have 1 container/pod, so I believe we should only keep `securityContext` there ATM.

Reply:
Removed